docs/brain/signals/order.md

```diff
@@ -8,12 +8,13 @@ An **order** signal is a word, or a sentence caught by the microphone and proces
 
 Other way to write an order, with parameters:
 
-| parameter           | required | default | choices                        | comment                                                |
-|---------------------|----------|---------|--------------------------------|--------------------------------------------------------|
-| text                | YES      |         |                                | The order to match                                     |
-| matching-type       | NO       | normal  | normal, strict, ordered-strict | Type of matching. See explanation bellow               |
-| stt-correction      | NO       |         |                                | List of words from the order to replace by other words |
-| stt-correction-file | NO       |         |                                | Same as stt-correction but load words from a YAML file |
+| parameter           | required            | default | choices                                     | comment                                                |
+|---------------------|---------------------|---------|---------------------------------------------|--------------------------------------------------------|
+| text                | YES                 |         |                                             | The order to match                                     |
+| matching-type       | NO                  | normal  | normal, strict, ordered-strict, not-contain | Type of matching. See explanation bellow               |
+| excluded_words      | only if not-contain |         |                                             | The words which are present in the order to prevent other orders to be execute |
```

Comment:
I hate being that guy, but... We used a dash for all other flag...

Reply:
Yep ... we should put a dash :) just to be consistent

Reply:
Ehm... not really sure what you guys mean by using a dash

Reply:
`excluded_words` --> `excluded-words`